README.md

```diff
@@ -1,51 +1,82 @@
 ## KiranaShop App
 
-### Local setup 
-- Clone [KiranaShop](https://github.com/ndedhia1210/KiranaShop) 
+### Local setup
+
+- Clone [KiranaShop](https://github.com/ndedhia1210/KiranaShop)
+
 ```
 https://github.com/ndedhia1210/KiranaShop.git
 ```
 
-- Build project locally 
+- Build project locally
+
 ```
 npm i
 ```
 
 - Run App locally
+
 ```
-npm start 
+npm start
 ```
+
 OR
+
 ```
-npx expo start   
+npx expo start
 ```
+
 Then press `w`
 
 - Visit following Url to test local instance
+
 ```
 http://localhost:19006
 ```
 
+### Directory structure
+
+```
+app   // contains all the source code
+    |-- api   // contains clients for APIs
+	|-- assets  // contains resources like images, animated gifs etc
+					|-- imgs
+					|-- animations
+					|-- icons
+	|-- auth   // contains code for authorization and authentication
+	|-- components   // contains building blocks for constructing views/screens
+	|-- config   // contains files to hold theme/setting/config variables
+	|-- hooks   // contains custom hooks
+	|-- navigation   // contains navigators and router for the app
+	|-- screens   // contains consolidated view of components. Screen corresponds to mockup in Figma.
+	|-- utility // contains utility files for async storage, logging, analytics etc
+```
```

Comment:
I was thinking about keeping directories little simple as we are just starting, I felt it didn't need to be more complex. I'm open to changing it if you think this would be better.

If I map my dirs to yours it would look like this -
services --> api
views --> components + screen

I didn't account for a store. So I can add that. 

Screen is nothing but the functional page/panel/view like how we have mocked up in Figma. Example CategoryList, LoginScreen, OrdersList, AccountScreen etc.

Reply:
Please find my comments in the same order
- Got it! I feel proposed file directory will improve our coding standards and we will end up creating our own app framework which we can follow in upcoming project. Let me know your thought. 
- Yes, you are right with mappings, will be good if you combine screen and components under view/components/screens any name of your choice. Feel free to give any name as per your preference, I am fine with it. 
- Store -> Cool
- Got the meaning of screens

Reply:
This is the only open item from this PR. Let me know if you want to address this in current PR or new PR? I am fine with anything.

Reply:
Addressed it in this PR itself so that we avoid missing it in follow-up PR.